src/components/header/Header.tsx

```diff
@@ -2,17 +2,25 @@ import style from './Header.module.sass';
 import logo from './logo.svg';
 import menu from './menu.svg';
 import search from './search.svg';
-
 /**
  * The header of the website.
  */
-export default function Header() {
+
+interface Props {
+  toggleSideNav: Function
```

Comment:
Don't use Function as a type. Use `() => void` instead.
Do you have eslint installed?

![image](https://github.com/talisraeli/Yeda/assets/30414693/7c606875-8fff-411d-af5b-62c8f0147601)